action/protocol/execution/testdata/send-eth.json

```diff
@@ -0,0 +1,34 @@
+{
+    "initBalances": [{
+        "account": "io1mflp9m6hcgm2qcghchsdqj3z3eccrnekx9p0ms",
+        "rawBalance": "1000000000000000000000000000"
+    }, {
+        "account": "io1rc2d2de7rtuucalsqv4d9ng0h297t63w7wvlph",
+        "rawBalance": "0"
+    }, {
+        "account": "io1zf69lmyzkkzlywwqzzggst45qupvx2cyye62fp",
```

Comment:
please remove the accounts which are not used in the test. Same for all other tests.